lib/cisco_node_utils/cmd_ref/ospf_area.yaml

```diff
@@ -38,6 +38,35 @@ filter_list_out:
   set_value: '<state> area <area> filter-list route-map <route_map> out'
   default_value: false
 
+nssa:
+  multiple: true
+  get_value: '/^area <area> nssa(.*)$/'
+  set_value: '<state> area <area> nssa <no_summary> <no_redistribution> <default_information_originate> <route_map> <rm>'
+
+nssa_def_info_originate:
```

Comment:
`/nssa_def_info_originate/nssa_default_originate/`

We already have bgp properties that follow this format so let's not deviate.